ChangeLog.md

```diff
@@ -8,6 +8,7 @@
   can now accept URLs with International Domain Names (IDN).
 * Changed {Kernel#try} to not silently ignore `SyntaxError` exceptions.
 * Documentation improvements.
+* Added {Ronin::Support::Binary::CTypes::OS::Android}.
```

Comment:
You probably should have branched off of the `1.1.0` branch which contains all new features for the next release. `1.0.1` has already been released. I could try manually merging this into the `1.1.0` branch with some edits.

Reply:
I didn't get aware of the existence of the `1.1.0` branch. Thank you for it.